Common/include/dual_grid_structure.hpp

```diff
@@ -1245,4 +1245,118 @@ class CVertex : public CDualGrid {
 
 };
 
+/*!
+ * \class CTurboVertex
+ * \brief Class for vertex definition for turbomachinery (equivalent to edges, but for the boundaries).
+ * \author S. Vitale
+ * \version 5.0.0 "Raven"
+ */
+class CTurboVertex : public CVertex {
```

Comment:
As I said in the description of the pull request, to impose Non reflecting BC and Mixing plane you need to access the boundary information on a span-wise and pitch wise ordered manner. Moreover, you need to convert the velocities in different frame of references  w.r.t. the turbomachinery architecture you are simulating. The turboVertex takes care of all of these needs.